src/language-js/print/type-parameters.js

```diff
@@ -171,7 +173,13 @@ function printTypeParameter(path, options, print) {
   }
 
   if (node.constraint) {
-    parts.push(" extends", indent([line, print("constraint")]));
+    const groupId = Symbol("constraint");
+    parts.push(
+      " extends",
+      group(indent(line), { id: groupId }),
```

Comment:
Change this line to `group(line, { id: groupId })` seems not changing the output.

Reply:
No, we need this `indent`. If we remove `indent` call, Prettier outputs bellow:

```js
// input
export type OuterType3<
  LongerLongerLongerLongerInnerType extends
    LongerLongerLongerLongerLongerLo.ngerLongerLongerOtherType
> = { a: 1 };

// output
export type OuterType3<
  LongerLongerLongerLongerInnerType extends
  LongerLongerLongerLongerLongerLo.ngerLongerLongerOtherType
> = { a: 1 };
```